runtime/binding-tcp/src/main/java/io/aklivity/zilla/runtime/binding/tcp/internal/TcpBindingContext.java

```diff
@@ -33,14 +35,16 @@
 final class TcpBindingContext implements BindingContext
 {
     private final Map<KindConfig, TcpStreamFactory> factories;
+    private final AtomicInteger capacity;
 
     TcpBindingContext(
         TcpConfiguration config,
         EngineContext context)
     {
+        this.capacity = new AtomicInteger(ENGINE_WORKER_CAPACITY.getAsInt(config));
```

Comment:
This is not shared across cores, so doesn't need to be atomic.
Perhaps `MutableInteger` instead?